conf/ihatemoney.cfg

```
@@ -1,10 +1,10 @@
 DEBUG = True
-SQLALCHEMY_DATABASE_URI = 'mysql+pymysql://ihatemoney:MY_MYSQL_PW@localhost/ihatemoney'
+SQLALCHEMY_DATABASE_URI = 'mysql+pymysql://__DB_USER__:__DB_PWD__@localhost/__DB_NAME__'
 SQLACHEMY_ECHO = DEBUG
-SECRET_KEY = "MY_SECRET_KEY"
+SECRET_KEY = "__SECRET_KEY__"
 
-MAIL_DEFAULT_SENDER = ("Budget manager", "MY_EMAIL")
-APPLICATION_ROOT='MY_PATH'
+MAIL_DEFAULT_SENDER = ("Budget manager", "__MAILS_SENDER__")
+__SUB_PATH_ONLY__APPLICATION_ROOT='__PATH__'
```

```suggestion
__SUB_PATH_ONLY__ APPLICATION_ROOT='__PATH__'
```
missing space made me think that was a full variable name

---

That should be tested.

The cfg file is a python script. Here will be the corresponding code :
```
blabla
 APPLICATION_ROOT='the_path'
blabla
```

See the space at the start of the line ? This will trigger a `IndentationError: unexpected indent`.

---

you're right. So long for readability :)